lms/static/sass/views/_learner-profile.scss

```diff
@@ -0,0 +1,183 @@
+// lms - application - learner profile
+// ====================
+
+// Table of Contents
+// * +Container - Learner Profile
+// * +Main - Header
+// * +Settings Section
+
+$profile-photo-dimension: 120px;
+
+.ui-loading-indicator {
+    @extend .ui-loading-base;
+
+    padding-bottom: $baseline;
+
+    // center horizontally
+    @include margin-left(auto);
+    @include margin-right(auto);
+    width: $baseline*5;
+}
+
+.wrapper-profile {
+    min-height: 200px;
+    background-color: $white;
+
+    .ui-loading-indicator {
+        margin-top: 100px;
+    }
+}
+
+.profile-self {
+    .wrapper-profile-field-account-privacy {
+        @include clearfix();
+        @include box-sizing(border-box);
+        margin: 0 auto 0;
+        padding: ($baseline*0.75) 0;
+        width: 100%;
+        background-color: $gray-l3;
+
+        .u-field-account_privacy {
+            @extend .container;
+            border: none;
+            box-shadow: none;
+        }
+
+        .u-field-title {
+            width: auto;
+            color: $base-font-color;
+            font-weight: $font-bold;
+            cursor: text;
+        }
+
+        .u-field-value {
+            width: auto;
+            @include margin-left($baseline/2);
+        }
+
+        .u-field-message {
+            @include float(left);
+            width: 100%;
+            padding: 0;
+            color: $base-font-color;
+        }
+    }
+}
+
+.wrapper-profile-sections {
+    @extend .container;
+}
+
+.wrapper-profile-section-one {
+
+    width: 100%;
+    display: inline-block;
+    margin-top: ($baseline*1.5);
+
+    .profile-photo {
+        @include float(left);
+        height: $profile-photo-dimension;
+        width: $profile-photo-dimension;
+        display: inline-block;
+        vertical-align: top;
+    }
+}
+
+.profile-section-one-fields {
+
+    float: left;
+    width: flex-grid(4, 12);
+    @include margin-left($baseline*1.5);
+
+    .u-field {
+        margin-bottom: ($baseline/4);
+        padding-top: 0;
+        padding-bottom: 0;
+        @include padding-left(3px);
+    }
+
+    .u-field-username {
+        margin-bottom: ($baseline*0.75);
+
+        .u-field-value {
+            font-size: 1.7em;
+            font-weight: 600;
+            line-height: 0.9em;
+        }
+    }
+
+    .u-field-title {
+        width: 0;
+    }
+
+    .u-field-value {
+        width: 200px;
```

Comment:
this is cutting off longer usernames. maybe add a wider width to the rule at line 103.